use { println, assert } from "std::debug";
use { panic, unreachable } from "std::lib";

pub fn main() -> int {
    assert(1 != 1, "1 != 1");

    return 0;
}

main();
